content/docs/command-reference/add.md

```diff
@@ -160,8 +160,10 @@ not.
   [remote storage](/doc/command-reference/remote) to transfer external target to
   (can only be used with `--to-remote`).
 
-- `-o <path>`, `--out <path>` - destination `path` for the transferred data (can
-  only be used with `--to-remote`).
+- `-o <path>`, `--out <path>` - destination `path` for the transferred data. If
+  used with `--to-remote`, the data will be transferred to the remote storage.
+  Else, it will be transferred [to the cache](#example-transfer-to-cache) and
+  will be linked to the workspace.
```

Comment:
Prob no need to explain to-remote again here. Please consider this text instead (below), and please move this option before `--to-remote`.

```suggestion
- `-o <path>`, `--out <path>` - destination `path` inside the workspace, to
  [transfer](#example-transfer-to-cache) an external target into, and track it
  in-place. Note that this can be combined with `--to-remote` to avoid storing
  the data locally, while still adding it to the project.
```